ccd-definition/ComplexTypes/DQ/RequestedCourt.json

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "ID": "RequestedCourt",
+    "ListElementCode": "claimantRequestedCourt",
```

Comment:
should this be applicant1RequestedCourt?

Reply:
I find it very confusing all this applicant vs claimant stuff. The business uses claimant etc and in my mind the data model should reflect the business terms. So you tell me 😅

Reply:
it's all about trying to align terminology across all the services. So even if the labels show the service specific nouns, the underlying tech attributes should (try) to be consistent.

Reply:
that's a recipe for confusion though right? you have a data model that doesn't align with the questions being asked from a service point of view

Reply:
I think everyone's aware that it can be confusing. But how else do we manage the point that each service talks about the same things in different ways? Can't keep having similar data for applicant, claimant appellant, etc.

Reply:
this is actually a label, so data isn't persisted into db anyway